salt/vector/install.sls

```diff
@@ -13,3 +13,9 @@ ensure_state_of_systemd_service_file:
   file.managed:
     - name: /etc/systemd/system/vector.service
     - source: salt://vector/files/vector.service
+  {% if salt.grains.get('init') == 'systemd' %}
+  cmd.wait:
+    - name: systemctl daemon-reload
+    - watch:
```

Comment:
You can change this to `onchanges`

Reply:
I think I was emulating another state; probably `salt/edx/gitreload.sls`. I'll check the usage for `onchanges` ...

Reply:
Changed in 7b3d921d